tests/test_upstream_versions.py

```diff
@@ -1356,11 +1357,26 @@ def test_update_upstream_versions_run_parallel_custom_sources(
     ) == ("source a", "source b")
 
 
-@mock.patch("conda_forge_tick.update_upstream_versions.RANDOM_FRAC_TO_UPDATE", new=1.1)
+@pytest.fixture
+def version_update_frac_always():
+    new_settings = settings()
+
+    new_settings.frac_update_upstream_versions = True
+    use_settings(new_settings)
+
+    yield
+
+    # revert to default settings
+    use_settings(None)
```

Comment:
this means we cannot stack multiple settings override fixtures on top of each other. do you think we need this feature eventually?